docs/digger/admin/monitoring.asciidoc

```diff
@@ -0,0 +1,12 @@
+== Monitor Digger
+Nagios is provisioned by default in the Digger Ansible playbook. The Nagios
+instance will perform a number of checks.
+
+* Ensure the macOS machine is running and accessible
```

Comment:
Would it be worth saying that we are ensuring that the macOS machine is accessible `to Jenkins`